src/instructlab/configuration.py

```diff
@@ -653,6 +653,11 @@ class _train(BaseModel):
         description="Base directory for organization of end-to-end intermediate outputs.",
     )
 
+    training_journal: str | None = Field(
+        default_factory=lambda: os.path.join(DEFAULTS.PHASED_DIR, "journalfile.yaml"),
```

Comment:
Can we use the run_id for the file name? or `journal-<uuid>.yaml`? So that we can run multiple phased trainings at the same time?

Reply:
This isn't currently a supported pattern. Moreover, to do so one would want to have a wholly different `phased_base_dir` I think, so IMO having this naming / location be static is desirable.

If one wanted to train multiple models on the same machine (e.g. on an 8xH100 machine, using GPUs 0-3 for one run and 4-8 for another) the wouldn't want to use the same multiphase checkpoint storage anyway.

Reply:
this can certainly be a feature in the future. right now, the journal can be renamed by passing a specific location / name for it, but it's supposed to be a hidden implementation detail from the user, minus being able to refer to it for other checkpoint scores if needed.